From 41e10abbe20b1114a7d319fd04fbe8ed2906536d Mon Sep 17 00:00:00 2001
From: Lars Kellogg-Stedman <lars@oddbit.com>
Date: Thu, 17 Sep 2026 20:16:22 -0400
Subject: [PATCH 1/4] Allow OPT_COLOUR=no to build successfully

Building with OPT_COLOUR=no would fail due to a header dependency bug:
header.c calls mx_go_input_cp() but does not include mx/go.h. The
color-enabled build accidentally hid this because mx/colour.h includes
mx/go.h when color support is enabled.
---
 src/mx/header.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/mx/header.c b/src/mx/header.c
index 543ba70a..add017d6 100644
--- a/src/mx/header.c
+++ b/src/mx/header.c
@@ -49,6 +49,7 @@
 #include "mx/cmd.h"
 #include "mx/compat.h"
 #include "mx/file-streams.h"
+#include "mx/go.h"
 #include "mx/mime-old.h"
 #include "mx/mime-probe.h"
 #include "mx/names.h"
-- 
2.55.0

